### PR TITLE
Simplify the operator install instructions

### PR DESCRIPTION
**Pull request by bpradipt**
Signed-off-by: Pradipta Banerjee <pradipta.banerjee@gmail.com>

### CODE REVIEW

**Review by ariel-adam — APPROVED** [review]

File: quickstart.md [hunk 1]
```diff
@@ -39,140 +39,51 @@ Confidential Containers is still maturing. See [release notes](./releases) for c
 
 You can enable Confidential Containers in an existing Kubernetes cluster using the Confidential Containers Operator.
 
-If you need to quickly deploy a single-node test cluster, you can use the [run-local.sh script](https://github.com/confidential-containers/operator/blob/main/tests/e2e/run-local.sh) from the operator test suite, which will setup a single-node cluster on your machine for testing purpose. This script requires `ansible-playbook`, which you can install on CentOS/RHEL using `dnf install ansible-core`, and the Ansible `docker_container` module, which you can get using `ansble-galaxy colection install community.docker`.
+:information_source: If you need to quickly deploy a single-node test cluster, you can use the [run-local.sh script](https://github.com/confidential-containers/operator/blob/main/tests/e2e/run-local.sh) from the operator test suite, which will setup a single-node cluster on your machine for testing purpose. This script requires `ansible-playbook`, which you can install on CentOS/RHEL using `dnf install ansible-core`, and the Ansible `docker_container` module, which you can get using `ansble-galaxy colection install community.docker`.
```
Comment (dcmiddle) [inline quickstart.md:42]: 1. break up long line to facilitate diffs.
2. fix ansible galaxy command

```suggestion
:information_source: If you need to quickly deploy a single-node test cluster, you can use the [run-local.sh script](https://github.com/confidential-containers/operator/blob/main/tests/e2e/run-local.sh) 
from the operator test suite, which will setup a single-node cluster on your machine for testing purpose. 
This script requires `ansible-playbook`, which you can install on CentOS/RHEL using `dnf install ansible-core`, and the Ansible `docker_container` module, which you can get using `ansible-galaxy collection install community.docker`.
```

**Review by dcmiddle — COMMENTED** [review]
Looks good. Just a cli typo.

**Review by fitzthum — APPROVED** [review]
This is fantastic. Thanks

**Review by sameo — COMMENTED** [review]
With @dcmiddle suggested fixes, this LGTM.

File: quickstart.md (hunk 1 above)
Comment (bpradipt) [reply quickstart.md:42]: I'll make the changes

File: quickstart.md [hunk 2]
```diff
@@ -189,14 +100,16 @@ kata-qemu-sev   kata-qemu-sev   9m55s
...
--- kata-qemu - same as kata
--- kata-qemu-tdx - using QEMU, with TDVF, and support for Intel TDX CC HW
--- * *TBD: we need to add the SEV runtimes as well* *
+- kata - standard kata runtime using the QEMU hypervisor including all CoCo building blocks for a non CC HW
+- kata-clh - standard kata runtime using the cloud hypervisor including all CoCo building blocks for a non CC HW
+- kata-clh-tdx - using the Cloud Hypervisor, with TD-Shim, and support for Intel TDX CC HW
+- kata-qemu - same as kata
+- kata-qemu-tdx - using QEMU, with TDVF, and support for Intel TDX CC HW
+- kata-qemu-sev - using QEMU, and support for AMD SEV HW
+
```
[the remark this answers is not in the document]
Comment (bpradipt) [reply quickstart.md:109]: Missed it. I think I can wait for the #56 to get merged and then rebase this one

**Comment by sameo** [comment]
@bpradipt #56 is merged, you can rebase now, thanks :)

**Comment by bpradipt** [comment]
> @bpradipt #56 is merged, you can rebase now, thanks :)

Done..

**Review by sameo — APPROVED** [review]

**Pull request merged**